lib/webhooks/webhooks.d.ts

```diff
@@ -1,10 +1,13 @@
-import { Request, Response } from 'express';
+import {IncomingHttpHeaders} from "http2";
 
-type Middleware = (
-  request: Request,
-  response: Response,
-  next: () => void
-) => any;
+interface Request {
+  protocol: string;
+  header(name: string): string | undefined;
+  headers: IncomingHttpHeaders;
+  originalUrl: string;
+  rawBody: any;
```

Comment:
Should be optional since it's artificial: https://github.com/twilio/twilio-node/pull/665
```
rawBody?: any;
```